wodles/aws/aws-s3.py

```diff
@@ -804,6 +804,17 @@ def reformat_msg(self, event):
             if field_to_cast in event['aws'] and not isinstance(event['aws'][field_to_cast], dict):
                 event['aws'][field_to_cast] = {'string': str(event['aws'][field_to_cast])}
 
+        if 'requestParameters' in event['aws']:
```

Comment:
You can do all this nested `if` in a single one:
```suggestion
        if event['aws'].get('requestParameters', {}).get('disableApiTermination') is not None:
```